website/docs/policy/getting-started.mdx

```diff
@@ -7,9 +7,8 @@ import TierLabel from "../_components/TierLabel";
 import CodeBlock from "@theme/CodeBlock";
 import BrowserOnly from "@docusaurus/BrowserOnly";
 
-# Getting started <TierLabel tiers="Enterprise" />
-
-This section introduces you to the Policy Profile and details the steps required to install it in Weave GitOps.
+# Getting Started <TierLabel tiers="Enterprise" />
+Enabling the Weave Policy Engine features in Weave GitOps is done by running the policy agent on the cluster. This section gives an overview of the steps required for installing and running the policy agent on the management and leaf clusters.
```

Comment:
I found the old text a bit misleading because this page doesnt really detail the steps required to install the profile, the user should first check the profile docs because it has all the necessary info about properly configuring the profile. THis is more of an overview. Also, since we use policy engine/agent/and profile interchangebly, I tried to set some context here and explain how they are all related so it wouldnt be confusing as the user moves from the intro into more detailed docs.